tox.ini

```diff
@@ -4,16 +4,16 @@ isolated_build = true
 
 [testenv]
 deps =
-    pytest==7.0.*
-    pytest-cov==3.0.0
-    coverage==6.2
+    pytest==7.1.*,>=7.1.2
```

Comment:
isn't that simply `>= 7.1.2`?

Reply:
ok, no: it prevents 7.2 and above to be used. I wonder if it isn't better to allow new major releases of the dependencies: the test environment will break occasionally, but so will it now once the specified versions become unsupported...

Reply:
I copied `pytest==7.1.*,>=7.1.2` from python-can. Back then this was the only combination of pytest and coverage, that worked for python 3.11.
 I updated these in the last commit, but i would keep them pinned to a specific version. It's annoying when the CI fails in PRs, just because there is a new pytest version.

Reply:
ok. I don't mind either.